packages/Datepicker/src/Datepicker.tsx

```diff
@@ -197,6 +197,14 @@ dateTimeOfDay;
         }
     };
 
+    const formattedMinDate = formatDate(minDate);
+    const formattedMaxDate = formatDate(maxDate);
+    const formattedSelectedDay = formatDate(selectedDay);
+    const minDateIsGreaterThanMaxDate = formattedMinDate && formattedMaxDate ? formattedMinDate > formattedMaxDate : false;
+
+    // Add a warning in the console to indicate that the min date is greater than the max date
+    minDateIsGreaterThanMaxDate && console.warn("minDate is greater than maxDate");
```

Comment:
Est-ce qu'on veut mettre un peu plus de contexte (genre "The prop minDate is greater than maxDate for the Datepicker" ou qqch comme ça)?